checker/raw_result.go

```diff
@@ -31,9 +31,48 @@ type RawResults struct {
 	WebhookResults              WebhooksData
 	MaintainedResults           MaintainedData
 	SignedReleasesResults       SignedReleasesData
+	FuzzingResults              FuzzingData
 	LicenseResults              LicenseData
 }
 
+// FuzzerName represents a fuzzing service.
+type FuzzerName int
+
+const (
+	// FuzzerNameCIFuzz is CIFuzz.
+	FuzzerNameCIFuzz FuzzerName = iota
```

Comment:
> I tend to re-use the type name to avoid name collision and to make it clearer what type it refers to in general. Shall I change that?

Upto you. Feel free to leave it as is. 

> I prefer enums because the linter will warn us if we forget to. There will be a stringify function when we need to print it, like in the json_raw_results.go file. Wdut?

Linter will still catch string enums if the switch case is not exhaustive. If you want to continue using integer however, I suggest adding a `String()` function. It'll make the enum <-> string conversion more robust (i.e will not depend on the caller).

```
func (f FuzzerName) String() string {
  switch f {
    // Linter ensures all integer values have a string name. 
  } 
}
```

Reply:
if the string are caught by linters, then I'm fine with strings.